src/main/java/com/cognite/beam/io/RequestParameters.java

```diff
@@ -274,7 +274,7 @@ public RequestParameters withItems(List<? extends Map<String, Object>> items) {
     @Deprecated
     public RequestParameters withItemExternalId(String externalId) {
         return toBuilder()
-                .setRequest(getRequest().withItemExternalId(externalId))
+                .setRequest(getRequest().withRootParameter("externalId", externalId))
```

Comment:
getRequest().withItemExternalIds(externalId) would be the correct target method.